01_prepare_host.sh

```diff
@@ -188,7 +188,48 @@ function configure_minikube() {
     minikube config set driver kvm2
     minikube config set memory 4096
 }
+
+#
+# Create Minikube VM and add correct interfaces
+#
+function init_minikube() {
+    #If the vm exists, it has already been initialized
+    if [[ "$(sudo virsh list --name --all)" != *"minikube"* ]]; then
+      # Loop to ignore minikube issues
+      while /bin/true; do
+        minikube_error=0
+        # Restart libvirtd.service as suggested here 
+        # https://github.com/kubernetes/minikube/issues/3566
+        sudo systemctl restart libvirtd.service
+        configure_minikube
```

Comment:
The `configure_minikube` function is two lines so it would be clearer to copy the lines here. This isn't a blocker for merging at all.